packages/ui/src/proposals/components/ProposalList/ProposalListItem.tsx

```diff
@@ -74,7 +77,19 @@ export const ProposalListItem = ({ proposal, isPast, memberId, isCouncilMember }
       </ToggleableItemInfo>
       <StageField>
         <TextSmall bold>{camelCaseToText(proposal.status)}</TextSmall>
-        <Tooltip tooltipText={checkStatus()}>
+        <Tooltip
+          tooltipText={
+            <>
+              {checkStatus()}{' '}
+              <TooltipExternalLink
```

Comment:
There is no need to insert an external link manually since this `Tooltip` component has the `tooltipLink...` props to handle this